fluss-flink/fluss-flink-common/src/test/java/com/alibaba/fluss/flink/source/metrics/FlinkSourceReaderMetricsTest.java

```diff
@@ -32,7 +33,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** Unit test for {@link com.alibaba.fluss.flink.source.metrics.FlinkSourceReaderMetrics}. */
-class FlinkSourceReaderMetricsTest {
+class FlinkSourceReaderMetricsTest extends FlinkTestBase {
```

Comment:
`FlinkSourceReaderMetricsTest` is not an integration test (`ITCase`) that doesn't rely on a fluss cluster and flink cluster, therefore, it doesn't need to extend `FlinkTestBase`. Setting up clusters is very heavy which should be avoided as much as possible to reduce the test time.